src/reports/citytax/report.ts

```diff
@@ -0,0 +1,102 @@
+    let rows: any[][] = [];
+
+    const _ = lodash();
+
+    const transactionsWithReservations = _.map(transactions, function (item) {
+        return _.merge(item, _.find(reservations, {'id': item.reference}));
+    });
+    const summarizedData = _(transactionsWithReservations)
+        .groupBy(value => value.source ?? value.channelCode)
+        .map((value, key) => {
+            const totalWithoutVat = _.sumBy(value, 'amount.amount') ?? 0;
+            const totalWithVat = (totalWithoutVat / 93 * 100) ?? 0;
+            const revenue = (totalWithVat / 5 * 100) ?? 0;
+            return {
+                channelCode: key,
+                cityTaxWithoutVat: totalWithoutVat,
+                cityTaxWithVat: totalWithVat,
+                netAccommodationRevenue: revenue
+            } as BerlinCityTaxRowItemModel;
+        })
+        .value();
+
+    for (const item of summarizedData) {
+        rows.push([item.channelCode, item.cityTaxWithoutVat, item.cityTaxWithVat, item.netAccommodationRevenue]);
+    }
+
+    //set headers
+    sheet.getRange(5, 1, 1, 4)
+        .setValues([['Channel Source', 'City Tax excl. VAT', 'City Tax Incl. VAT', 'Net accommodation revenue']])
+        .setFontWeight("bold");
+    if (rows.length <= 0) return;
+    const numberOfCols = rows[0].length;
+
+    //set data
+    sheet
+        .getRange(6, 1, rows.length, numberOfCols)
+        .setValues(rows)
+
+    //format data
+    sheet.getRange(6, 2, rows.length, numberOfCols - 1)
+        .setNumberFormat("0.00");
+}
+
+function createSheetWithReportInfo(city: string, property: string, endDate: string, startDate: string) {
+    const activeSpreadsheet = SpreadsheetApp.getActiveSpreadsheet();
+    const newSheetName = `citytax_${city}_${property}_${endDate}`;
+    let datasheet = activeSpreadsheet.getSheetByName(newSheetName);
+    if (!datasheet) datasheet = activeSpreadsheet.insertSheet().setName(newSheetName);
+    datasheet.clear();
+    datasheet.clearFormats();
+    activeSpreadsheet.setActiveSheet(datasheet);
+
+    const firstCell = datasheet.getRange(1, 1);
+    firstCell.setValue("City Tax Report").setFontSize(18);
+    datasheet
+        .getRange(2, 1)
+        .setValue(`for property ${property} from ${startDate} to ${endDate}`);
+    datasheet
+        .getRange(3, 1)
+        .setValue("Executed: " + formattedExecutionTime());
+
+    return datasheet;
+}
+
+function getTransactions(property: string, transactionStartDate: Date, transactionEndDate: Date, startDate: string, endDate: string) {
+    let transactions = getAccountTransactions(property, 'CityTax_Reduced:7.00', transactionStartDate, transactionEndDate);
+    const reportDaysList = getDates(startDate, endDate).map(d => d.toISOString().slice(0, 10));
+    return transactions.filter(transaction => transaction.command == "PostCharge" && reportDaysList.includes(transaction.date));
```

Comment:
the same for `return` let's add an empty line (like it's done in the function above)